### PR TITLE
Full Function

### LINKED ISSUE #6: Dimen File
**Is your feature request related to a problem? Please describe.**
Dimensions are repeated for the same use case across the layout files.

**Describe the solution you'd like**
Create a dimen file to store these standard dimensions

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #7: Style and Theming
**Is your feature request related to a problem? Please describe.**
Colors are hardcoded across project and support for color changes from day to night mode should be considered.
There should also be standard styles for duplicated views.

**Describe the solution you'd like**
[] Move all hardcoded colors to colors file
[] Create custom styles for views
[] Add style/colors to both theme folders (day/night)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Jherscu**
# Description

Refine UI, Logic, and:
- Add dimen, style, and theme resources
- Update icons
- Cover edge cases

Closes #6
Closes #7

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Jherscu** [comment]
@j-roskopf All done!

File: app/src/main/java/com/example/tictactoe/MainActivity.kt [hunk 1]
```diff
@@ -43,4 +48,30 @@ class MainActivity : AppCompatActivity() {
...
+    override fun onWinningDialogPositiveClick(
+        dialog: DialogFragment,
+        viewModel: TicTacToeViewModel,
+        fragment: GameFragment
+    ) {
+        // Play again with the same names
+        viewModel.resetGameState()
+        // Refresh fragment from scratch so the UI reflects the cleared data
+        fragment.parentFragmentManager.beginTransaction()
+            .detach(fragment)
```
Comment (j-roskopf) [inline app/src/main/java/com/example/tictactoe/MainActivity.kt]: could a detach / attach just be replaced with a `replace` call?

**Review by j-roskopf — APPROVED** [review]
love the icon :) it's looking really good! I wish I had more to say, but in general things were a of a good quality and already implemented a lot of best practices

File: app/src/main/java/com/example/tictactoe/MainActivity.kt (hunk 1 above)
Comment (Jherscu) [reply app/src/main/java/com/example/tictactoe/MainActivity.kt]: I'll switch it out and give it a go! Thanks!

Comment (Jherscu) [reply app/src/main/java/com/example/tictactoe/MainActivity.kt]: Just updated the code! As long as I am using the safeArgs plugin I have to refresh it via one of their actions, not with a replace call. My original version was a bit hack-y in retrospect. Should be good now!

**Pull request merged**